stable/buzzfeed-sso/templates/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "buzzfeed-sso.fullname" . }}
+  labels:
+    app: {{ template "buzzfeed-sso.name" . }}
+    chart: {{ template "buzzfeed-sso.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  upstream_configs.yml: |-
+  {{- range .Values.ingress.hosts }}
+    - service: {{ .serviceName }}
```

Comment:
I think it would make sense to add support for configuring other 'upstream_config' parameters as well, see https://github.com/buzzfeed/sso/blob/f437f237ac977201f15868601c9bc0e9dff11f40/docs/sso_config.md#proxy-config

Reply:
Good point. Even if we will need to repeat the domain again, I have separated the `ingress.hosts` configuration from the `upstreams`. Now we have an `upstreams` list of configurations that exactly fits with the url you just posted.

This should give complete flexibility about how the upstreams are configured